django/exercises/views.py

```diff
@@ -48,6 +51,44 @@ def get_context_data(self, **kwargs):
         return context
 
 
+# Export Data: Student Scores
+
+
+class ExportDataStudentScoresOptionsTemplateView(LoginRequiredMixin, TemplateView):
+    """
+    Class-based view for export data: student scores options template
+    """
+    template_name = 'exercises/exportdata-studentscores-options.html'
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+
+        # Return all classes if user is an admin
+        if self.request.user.role.name == 'admin':
+            context['classes'] = models.SchoolClass.objects.all()
+        elif self.request.user.role.name == 'teacher':
+            context['classes'] = models.SchoolClass.objects.filter(user__id=self.request.user.id)
```

Comment:
Is 'user__id' correct (double underscore)?

Reply:
yeah, so double underscores are used to access fields of related data. so in this case user is a foreign key and I want to get its Id so it's user__id ... e.g. let's say that user has a role and I want to get the name of that role you'd use double underscore again, so user__role__name :)